apps/src/levelbuilder/level-editor/EditChildLevelSettings.jsx

```diff
@@ -0,0 +1,161 @@
+//Create a boilerplate for the EditChildLevelSettings component
+//This component will be used to edit the settings of any child levels
+//The component will take in the following props:
+//Child levels: an array of objects representing the child levels
```

Comment:
Not worth fixing style nit for future ref: usually put a space after `// Before starting the comment`